File: 351001/shaminko/module-discussion/src/main/java/by/shaminko/distcomp/DistCompApplication.java
package by.shaminko.distcomp;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.Bean;
import org.springframework.web.reactive.function.client.WebClient;

@SpringBootApplication
public class DistCompApplication {

	public static void main(String[] args) {
		SpringApplication.run(DistCompApplication.class, args);
	}
	@Bean
	public WebClient webClient() {
		return WebClient.builder()
				.baseUrl("http://localhost:24130")
				.build();
	}
}

File: 351002/Abrashin/publisher/src/main/kotlin/com/example/distcomp/DistCompApplication.kt
package com.example.distcomp

import org.springframework.cache.annotation.EnableCaching
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
@EnableCaching
class DistCompApplication

fun main(args: Array<String>) {
    runApplication<DistCompApplication>(*args)
}

